modules/contentbox/modules/contentbox-ui/views/adminbar/adminbarCSS.cfm

```diff
@@ -0,0 +1,190 @@
+        transition: transform 1s ease;
+        -moz-transition: transform 1s ease;
+        -webkit-transition: transform 1s ease;
+    }
+    .slide_out .cb-adminbar__toggle {
+        box-shadow: 2px 2px 4px rgb(0 0 0 / 20%); 
+        transform: translateY( 100% );
+    }
+    .slide_out .cb-adminbar__toggle .svg-cheveron {
+        transform: rotate( 180deg );
+    }
+    .cb-adminbar__content {
+        display: flex;
+    }
+    .cb-adminbar__brand {
+        padding: 0.5rem;
+        margin-left: 40px;
+    }
+    .cb-adminbar__menu {
+        display: flex;
+        list-style: none;
+        margin: 0px;
+    } 
+    .cb-adminbar__icon {
+        display: inline-block;
+        margin-right: 5px;
+        width: 14px;
+    }
+    .cb-adminbar__brand .cb-adminbar__icon{
+        width: 20px;
+        margin-right: 0px; 
+    }
+    .cb-adminbar__dropdown-toggle {
+        padding: 0.5rem
+    }
+    .cb-adminbar__menu-item a, .cb-adminbar__menu-item button, .cb-adminbar__brand {
+        color: ##fff;
+        display: inline-block;
+        line-height: 25px;
+        padding: 0.5rem;
+    }
+    
+    .cb-adminbar__brand:hover, 
+    .cb-adminbar__brand:focus, 
+    .cb-adminbar__menu-item a:hover, 
+    .cb-adminbar__menu-item a:focus,
+    .cb-adminbar__menu-item button:hover,
+    .cb-adminbar__menu-item button:focus,
+    .cb-adminbar__dropdown.active  .cb-adminbar__dropdown-toggle {
+        background: ##222;
+        color: ##8fc73e;
+        text-decoration: none;
+    }
+    .cb-adminbar__dropdown-menu ul {
+        border-top: 1px solid ##3f3f3f;
+        list-style: none;
+        margin: 0px;
+        margin-top: 10px;
+        padding: 0px;
+    }
+    .cb-adminbar__dropdown-menu ul > li a {
+        color: ##fff;
+        display: block;
+        padding: 0.5rem;
+    }
+    .cb-adminbar__dropdown-menu ul > li a:hover {
+        background-color: ##222;
+        color: ##8fc73e;
+        text-decoration: none;
+    }
+    .cb-adminbar__dropdown.active .cb-adminbar__dropdown-menu {
+        display: block;
+    }
+    .cb-adminbar__badge{
+        border-radius: 50rem;
+        float: right;
+        padding: 0.35em 0.65em;
+        font-size: .75em;
+        font-weight: 700;
+        line-height: 1;
+        text-align: center;
+        white-space: nowrap;
+        vertical-align: baseline;
+    }
+    .cb-adminbar__badge.bg-success {
+        background-color: ##8fc73e;
+        color: ##000;
+    }
+    .cb-adminbar__badge.bg-danger {
+        background-color: rgb(220, 53, 69);
+    }
+    .cb-adminbar__badge.bg-warning {
+        background-color: rgb(255, 193, 7);
+        color: ##000;
+    }
+    .cb-adminbar__dropdown-menu {
+        position: absolute;
+        background: rgba(34,34,34,0.90);
+        -webkit-border-radius: 0px 0px 4px 4px;
+        border-radius: 0px 0px 4px 4px;
+        max-width: 250px;
+        padding: 0.5rem; 
+        z-index: 1000;
+        display: none;
+    }
+    .publisher img {
```

Comment:
I think we should namespace this with the `cb-adminbar` prefix or it could collide with the theme css

Reply:
Agreed.  I actually removed "publisher," and used a more generic prefixed class to style the image.